net/redeemer.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package net;
+
+service TicketRedeemer {
+    rpc QueueTicket(Ticket) returns (google.protobuf.Empty) {}
```

Comment:
Stub out an empty response type here, rather than using `Empty`.  We should be acking the request anyway to ensure the ticket was actually written to disk. This also gives us room to add fields to the response if necessary without breaking compatibility. Something like:

```proto
message QueueAck{
}
rpc QueueTicket(Ticket) returns (QueueAck) {}
```

Reply:
I think I replied to this somewhere else as well. 

`empty.Empty` is designed exactly for this case and will not break backwards compatibility of the wire protocol. 

https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#empty

gRPC will return an error and internal error status code when we run into an error throughout the operation. If there is no error then an empty object and status OK will be returned.

Reply:
> will not break backwards compatibility of the wire protocol.

Are you certain of that? The documentation doesn't claim anything about back compatibility. Some quick googling indicates this could break, and the use of `Empty` is generally not encouraged:

https://stackoverflow.com/questions/50993815/is-google-protobuf-empty-dangerous-for-backwards-compatibility

By the author of Protocol Buffers v2 and Cap'n Proto: https://stackoverflow.com/questions/31768665/can-i-define-a-grpc-call-with-a-null-request-or-response/31772973#comment51526939_31772973

Reply:
I think what is meant in the first SO thread is that it can surely break backwards compatibility for the generated stubs but the behaviour between using `empty.Empty` or an empty defined message remains the same so it changes nothing over the wire (both are empty objects in gRPC). 

I'm fine with defining an empty message explicitly though if it aleviates concerns and I can see why that type safety  for generated stubs could be relevant in the future.

Reply:
fixed in 258e02747160df22b465787adfd245ce9ed02a65